code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -686,9 +686,9 @@
 /obj/item/spellbook/attack_self(mob/user)
 	if(!owner)
 		to_chat(user, span_notice("You bind the spellbook to yourself."))
-		owner = user
+		owner = user.mind
```

Comment:
owner is typecasted as `var/mob/living/carbon/human` and user.mind is `/datum/mind`.
Retype owner, dmdoc it, and make sure to check the rest of the code for what changing owner might matter for.